cli/driver/Cargo.toml

```diff
@@ -25,3 +25,4 @@ log.workspace = true
 pretty_env_logger.workspace = true
 inquire = "0.6"
 const_format = "0.2"
+cargo_metadata = "0.15"
```

Comment:
Can you move that dependency to the root `Cargo.toml`, and then use `cargo_metadata.workspace = true` here and in `test-harness/Cargo.toml`?